js/work-section.js

```diff
@@ -0,0 +1,302 @@
+const htmls = document.getElementsByTagName('html')[0];
+const cardContainer = document.querySelector('.card-container');
+const popupContainer = document.querySelector('main article');
+
+// Array of data that is implemented in the pop-up window
+const projects = [
+  {
+    id: 'project_1',
+    images: {
+      img: './images/Snapshoot-Portfolio-mobile1@2x.png',
+      cancelPop: './images/icons/Disabled@2x.png',
+      altText: 'project 1 preview image',
+    },
+    name: 'Multi-Post Stories Gain+Glory_1',
+    heading: 'Keeping track of hundreds of components_1',
+    languages: [
+      'Ruby on Rails',
+      'Css',
+      'JavaScript',
+      'html',
+    ],
+    technologies: [
+      'Codekit',
+      'GitHub',
+      'Javascript',
+      'Bootstrap',
+      'Terminal',
+      'Codepen',
+    ],
```

Comment:
- [ ] I do not see this object in any place when I did opened the First card on mobile view see the screenshot 👇

![onm](https://user-images.githubusercontent.com/61562626/131745515-5ff6a75e-43a6-4427-a535-952b8074eb6c.jpeg)

Reply:
It seems that The changes you have requested are supposed to be implemented in the mobile version and I have already implemented them,
In the mobile version, we only have three languages that are displayed and it's only in the desktop version where we have all the technologies above in the object displayed as in the Figma design and I have already implement all these, can you please check out this once again?